en/docs/learn/examples/transport-examples/tcp-transport-examples.md

```diff
@@ -39,49 +39,62 @@ The following are the transport receiver properties.
 
 The following proxy service splits the message by a character. It receives a message with an empty body, which it will forward to the HTTP endpoint after enriching the body with the symbolic value "`IBM`".
 
-```xml
-<proxy xmlns="http://ws.apache.org/ns/synapse" name="TCPProxy" 
-              transports="tcp" 
-              startOnLoad="true" 
-              trace="disable"> 
-          <description/> 
-          <target> 
-             <inSequence> 
-                <property name="symbol" value="IBM" scope="default" type="STRING"/> 
-                <enrich> 
-                   <source type="inline" clone="true"> 
-                      <m:getQuote xmlns:m="http://services.samples"> 
-                         <m:request> 
-                            <m:symbol>?</m:symbol> 
-                         </m:request> 
-                      </m:getQuote> 
-                   </source> 
-                   <target type="body"/> 
-                </enrich> 
-                <enrich> 
-                   <source type="property" clone="true" property="symbol"/> 
-                   <target xmlns:m="http://services.samples" xpath="//m:getQuote/m:request/m:symbol"/> 
-                </enrich> 
-                <log level="full" separator=","/> 
-                <header name="Action" value="urn:getQuote" />
-                <call> 
-                   <endpoint> 
-                      <address uri="http://localhost:9000/services/SimpleStockQuoteService" format="soap11"/> 
-                   </endpoint> 
-                </call> 
-                <log level="full"/> 
+=== "Proxy Service"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="TCPProxy" startOnLoad="true" transports="tcp" xmlns="http://ws.apache.org/ns/synapse">
+        <target>
+            <inSequence>
+                <property name="symbol" scope="default" type="STRING" value="IBM"/>
+                <enrich description="">
```

Comment:
empty description.

Reply:
by default when we create an enrich mediator it adds `description=""` even we didn't provide a description